torch/__init__.py

```diff
@@ -24,7 +24,8 @@
     USE_RTLD_GLOBAL_WITH_LIBTORCH, USE_GLOBAL_DEPS
 # TODO(torch_deploy) figure out how to freeze version.py in fbcode build
 if sys.executable == 'torch_deploy':
-    __version__ = "torch-deploy-1.8"
+    # Add major.minor suffix to torch-deploy version
+    __version__ = "torch-deploy-" + ".".join(torch.version.__version__.split('.')[:2])
```

Comment:
i think this won't work in fbcode, but if it did, i'd be happy to merge.

the issue is that in the torch deploy build, torch.version isn't available, since it is an autogenerated file in buck that we didn't figure out a good way to include in the frozen python distribution.  It can be solved, just wasn't dealt with yet.

No task/ETA as far as I'm aware, you can file one on me.

@zdevito any other details or ideas on how to deal with this?

Reply:
Why does it need a version number to begin with? I.e. why "torch-deploy" would not do?

Reply:
i actually had it as 'torch-deploy' or similar to begin with.  I believe @zdevito changed to a numeric version string to placate some models or use cases that crashed if the version wasn't correctly formed.  But i'm not sure about the details there.